doi/src/main/webapp/paths/doi-mint.yaml

```diff
@@ -0,0 +1,20 @@
+post:
+  tags:
+    - "instances"
+  summary: "Finalize a DOI instance"
+  description: "A multi-step process that freezes the DOI data from further changes, registers the DOI to DataCite and makes the DOI findable from DataCite. The DOI status indicates the progress of the finalize process."
+  operationId: "mintDOI"
+  parameters:
+    - $ref: ../components/parameters/doi-doinum.yaml
+  responses:
+    200:
```

Comment:
The result of a successful publish to DataCite is a 303 with a redirect to the DOI.